flake.nix

```diff
@@ -33,51 +34,42 @@
         pkgs = import nixpkgs { inherit system; };
 
         easy-ps = import inputs.easy-ps { inherit pkgs; };
-        pursBridgeHsProjectFor = system: import ./nix/haskell.nix {
+
+        pursBridgeHsProject = import ./nix/haskell.nix {
           inherit src system pkgs easy-ps;
           inputs = inputs // inputs.bot-plutus-interface.inputs;
           extraSources = inputs.bot-plutus-interface.extraSources;
         };
-        pursBridgeFlakeFor = system: (pursBridgeHsProjectFor system).flake { };
-        cq = import ./nix/code-quality.nix { projectName = ""; inherit pkgs easy-ps; };
+
+        pursBridgeFlake = pursBridgeHsProject.flake { };
+
+        cq = import ./nix/code-quality.nix {
+          projectName = "purescript-bridge-code-quality";
+          inherit pkgs easy-ps;
+        };
+
         fileCheckers = cq.checkers pkgs;
 
         # plutus-ledger-api Purescript typelib
         ledgerTypelib = import ./nix/purescript-bridge-typelib.nix {
           inherit pkgs;
-          purs = easy-ps.purs-0_14_5; # TODO: Extract the purs version as a param and share across
-          cli = (pursBridgeHsProjectFor system).getComponent "purescript-bridge:exe:cli";
+          # TODO: Extract the purs version as a param and share across
+          purs = easy-ps.purs-0_14_5;
+          cli = pursBridgeHsProject.getComponent "purescript-bridge:exe:cli";
         };
-      in
-      {
+
+      in {
         # Useful attributes
-        inherit pkgs ledgerTypelib;
-        pursBridgeFlake = pursBridgeFlakeFor system;
-        pursBridgeHsProjectFor = system: import ./nix/haskell.nix {
-          inherit src system pkgs easy-ps;
-          inputs = inputs // inputs.bot-plutus-interface.inputs;
-          extraSources = inputs.bot-plutus-interface.extraSources;
-        };
```

Comment:
I wonder if this was the issue here? @bladyjoker

Reply:
I needed it here for convenience as I was debugging in `nix repl`